src/vm/vm.cpp

```diff
@@ -79,24 +80,28 @@ void Init() {
       OpcodeTableSystem = OpcodeTableSystem_Darling;
       OpcodeTableGraph = OpcodeTableGraph_Darling;
       OpcodeTableUser1 = OpcodeTableUser1_Darling;
+      OpcodeTableGraph3D = nullptr;
```

Comment:
Let's move this initialization up to where these are declared (line 50 in this file) and don't explicitly set it for each set that doesn't have 3D.

Reply:
Done.